Pipfile

```diff
@@ -28,6 +28,7 @@ coverage = "==7.2.2"
 pytest-cov = "==4.0.0"
 ipython = "==8.12.0"
 pytest-asyncio = "==0.21.0"
+rpy2 = "==3.5.10"
```

Comment:
It sounds like this is going away in a later PR - can we get rid of it here so folks using main don't have to install it and then almost immediately uninstall it?

Reply:
right. that's why I added to `dev-packages` so that tests can pass. Do you prefer to completely remove it and the import from comput_genome_attris script?

Reply:
yeah let's get rid of it unless there's a really compelling reason to keep it

Reply:
👍

Reply:
:+1: